app/views/audit_mailer/summary.html.erb

```diff
@@ -0,0 +1,31 @@
+<body style="font-family: 'Helvetica Neue', Helvetica, Arial, sans-serif; color: #3f3f3f; background-color: #f1f1f1; padding: 10px 24px">
+<h2 style="font-weight: normal; text-transform: uppercase; font-size: 120%;"><%= _("<b>Foreman</b> audit summary").html_safe %></h2>
+<ul style="list-style-type: none;">
+  <% if @audits.size > 0 %>
+    <%= n_('Displaying %{num_audits} of %{total_audits} audit',
+           'Displaying %{num_audits} of %{total_audits} audits', @count) % {:num_audits => @limit, :total_audits => @count} %>
```

Comment:
I don't think `@limit` makes sense here, shouldn't it be the number of values in `@audits`?  e.g. "Displaying 10 of 1 audit" makes no sense